i18n/de.toml

```diff
@@ -142,9 +142,9 @@ pvProduction = "Erzeugung"
 selfConsumption = "Eigenverbrauch"
 
 [main.loadpoint]
-avgPrice = "Ø Preis"
+avgPrice = "⌀ Preis"
```

Comment:
Warum die Änderung- aus optischen Gründen?

Reply:
@andig

Nein, hier geht es (mir) nicht um die Optik, sondern um die Semantik:

- Zurzeit wird leider der Buchstabe Ø (https://de.wikipedia.org/wiki/%C3%98) verwendet, um Durchschnittswerte zu kennzeichnen. Das ist jedoch semantisch falsch.
- Das Durchschnittszeichen ⌀ (https://de.wikipedia.org/wiki/Durchmesserzeichen) wäre hier semantisch korrekt.
- Je nach (Betriebs-)System können Ø und ⌀ zwar sehr ähnlich oder sogar gleich aussehen. Dennoch, der sematische Unterschied zwischen den beiden Zeichen bleibt immer bestehen, auch wenn man ihn vielleicht nicht (auf Anhieb) sehen kann.

Das Durchschnittszeichen ⌀ sollte natürlich einheitlich in allen unterstützten Sprachen (anstelle des Buchstabens Ø) verwendet werden.

Reply:
> Zurzeit wird leider der Buchstabe Ø (https://de.wikipedia.org/wiki/%C3%98) verwendet, um Durchschnittswerte zu kennzeichnen. Das ist jedoch semantisch falsch.

Das Thema habe ich jetzt in #8865 für alle Sprachen adressiert.